src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
```

Comment:
Using static import for methods is generally not recommended as it can make the code less readable. It would be better to create an instance of the Lottery class and call the method on that instance.